packages/components/psammead-topic-tags/package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@bbc/psammead-topic-tags",
+  "version": "1.0.0",
```

Comment:
The package(-lock).json version's should match the changelog entry.

Reply:
Thanks for the comments @simonsinclair! I'll get started working through them 👍 Can't believe I forgot to change the package jsons version number 😂